README.md

```
@@ -1 +1,66 @@
+| ├── ClassifyPetDisease.ipynb -> 이름이 중복되면 수정
+│ └── model.pth
+|
+├── push_registry.py # Azure Model Registry 업로드 코드
+└── ...
+```
+
+## 협업 가이드
+
+### 0. 환경 설정
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)